### PR TITLE
changed zip url for github

### PR DESCRIPTION
**Pull request by utkarsh-lambdatest**
# Description

Replaced zip download link with API link

Note: **Support for Github apps**

Fixes # ([GitHub Apps support](https://github.com/LambdaTest/test-at-scale/issues/137))

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [X] Ran post and pre-merge jobs

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 NUCLEUS_DOCKER_FILE ?= ./build/nucleus/Dockerfile
-NUCLEUS_IMAGE_NAME ?= lambdatest/nucleus:latest
+NUCLEUS_IMAGE_NAME ?= nucleus
```
Comment (VikrantKS) [inline Makefile:2]: why this change ?

**Review by palashmarantas — DISMISSED** [review]

**Review by VikrantKS — APPROVED** [review]

**Review by VikrantKS — APPROVED** [review]
